app/upgrades/xenon/upgrade.go

```diff
@@ -0,0 +1,86 @@
+package xenon
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	florinkeeper "github.com/noble-assets/florin/x/florin/keeper"
+	halokeeper "github.com/noble-assets/halo/x/halo/keeper"
+)
+
+var (
+	// TODO: Verify denom metadata
+
+	eureMetaData = banktypes.Metadata{
```

Comment:
Just a style choice here

```suggestion
	eureMetadata = banktypes.Metadata{
```